drivers/homeless_summary_report/spec/models/report_context.rb

```diff
@@ -0,0 +1,55 @@
+###
+# Copyright 2016 - 2022 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+RSpec.shared_context 'report context', shared_context: :metadata do
+  def default_setup_path
+    'drivers/homeless_summary_report/spec/fixtures/files/default'
+  end
+
+  def default_filter
+    {
+      start: Date.parse('2022-01-01'),
+      end: Date.parse('2022-12-31'),
```

Comment:
Do we care that this is currently in the future?  (I'm guessing that won't affect outcome later, but just checking)

Reply:
It shouldn't -- I just picked a date range to use in the fixture, and matched it in the filter.